static/app/views/issueDetails/groupEventDetails/groupEventDetails.spec.tsx

```diff
@@ -502,4 +572,67 @@ describe('Platform Integrations', () => {
 
     expect(componentsRequest).toHaveBeenCalled();
   });
+
+  describe('ANR Root Cause', () => {
+    beforeEach(() => {
+      MockApiClient.clearMockResponses();
+    });
+    it('shows anr root cause', async () => {
+      const {organization} = initializeOrg();
+      const props = makeDefaultMockData({
+        ...organization,
+        features: ['anr-improvements'],
+      });
+      mockGroupApis(
+        props.organization,
+        props.project,
+        props.group,
+        props.event,
+        mockedTrace(props.project)
+      );
+      const routerContext = TestStubs.routerContext();
+      await act(async () => {
+        render(<TestComponent group={props.group} event={props.event} />, {
+          organization: props.organization,
+          context: routerContext,
+        });
+        await tick();
+      });
+
+      expect(
+        screen.getByRole('heading', {
+          name: /suspect root issues/i,
+        })
+      ).toBeInTheDocument();
+      expect(screen.getByText('File IO on Main Thread')).toBeInTheDocument();
+    });
+
+    it('does not render root issues section if related perf issues do not exist', async () => {
+      const {organization} = initializeOrg();
+      const props = makeDefaultMockData({
+        ...organization,
+        features: ['anr-improvements'],
+      });
+      const trace = mockedTrace(props.project);
+      mockGroupApis(props.organization, props.project, props.group, props.event, {
+        ...trace,
+        performance_issues: [],
+      });
+      const routerContext = TestStubs.routerContext();
+      await act(async () => {
```

Comment:
@shruthilayaj do you know why we have to do this? It's pretty weird to have to wrap a render with an act

Reply:
Uhh I just copied one of the existing tests and added stuff I needed